### PR TITLE
Add MarqueeLabel

### PR DESCRIPTION
**Pull request by MaxHasADHD**

### CODE REVIEW

File: CocoaGrapes/ViewController.swift [hunk 1]
```diff
@@ -7,13 +7,55 @@
...
+        label.text = "According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees don't care what humans think is impossible."
+        label.animationCurve = .easeInOut
+        label.trailingBuffer = 10
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.isUserInteractionEnabled = true
+        label.holdScrolling = true
+        self.view.addSubview(label)
+        
+        // Tap gesture
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(labelWasTapped))
```
Comment (petey3) [inline CocoaGrapes/ViewController.swift]: There is a way to do this built into MarqueeLabel

Comment (petey3) [inline CocoaGrapes/ViewController.swift]: There is a way to enable tapping to scroll built into MarqueeLabel, check the documentation

File: Pods/MarqueeLabel/LICENSE [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Copyright (c) 2011-2016 Charles Powell
```
Comment (petey3) [inline Pods/MarqueeLabel/LICENSE]: Looks like you are missing a .gitignore, so the entire pods are being added

**Review by petey3 — CHANGES_REQUESTED** [review]

**Comment by MaxHasADHD** [comment]
Fixed

**Pull request closed without merging**